### PR TITLE
add sdk-start script and temp code in api for sdk call

### PR DESCRIPTION
**Pull request by FaizChishtie**

### CODE REVIEW

**Comment by FaizChishtie** [comment]
Add callback in app.tsx to pass to api for render

File: src/App.tsx [hunk 1]
```diff
@@ -46,12 +47,16 @@ class App extends React.Component<IAppProps, IAppState>{
     microsoftTeams.registerOnThemeChangeHandler( this.props.onThemeChange );
   }
 
+  public handleDataOnChange = ( data: ICard[] ): void => {
```
Comment (sachingupta) [inline src/App.tsx]: rename this function to onResults

File: src/api/api.ts [hunk 2]
```diff
@@ -1,18 +1,23 @@
 import { default as jsonData } from './generated.json';
-import { ICard } from '../api/api.interface';
+import { ICard, IPreviewCard } from '../api/api.interface';
+import * as microsoftTeams from '@microsoft/teams-js';
 
-export const getResults = ( query: string ): ICard[] => {
+export const getResults = ( query: string, callback: ( response: ICard[] ) => void ) => {
```
Comment (sachingupta) [inline src/api/api.ts]: rename callback to onResults

**Comment by FaizChishtie** [comment]
fix pr changes above

**Pull request merged**